test/test_importer.py

```diff
@@ -414,7 +414,7 @@ class ImportTarTest(ImportZipTest):
     def create_archive(self):
         (handle, path) = mkstemp(dir=self.temp_dir)
         os.close(handle)
-        archive = TarFile(path, mode='w')
+        archive = TarFile(py3_path(path), mode='w')
```

Comment:
just that they were part of the overall issues with archive files, so i wanted to commit that as a whole change